src/towncrier/newsfragments/482.feature.rst

```diff
@@ -0,0 +1,5 @@
+If no filename is given when doing ``towncrier`` create, interactively ask for the issue number and fragment type (and then launch an interactive editor for the fragment content).
+
+Now by default, when creating a fragment it will be appended with the ``filename`` option's extension (unless an extension is explicitly provided). For example, ``towncrier create 123.feature`` will create ``news/123.feature.rst``. This can be changed in configuration file by setting `add_extension = false`.
```

Comment:
Not sure if this is feature, or a backward compatiblity break :)

Reply:
It's a change in the filename but since the file is parsed by towncrier in exactly the same way, I don't think it's really backwards incompatible

Reply:
OK. Thanks. Fine by me.

I am not sure that adding release notes in this way, in a single fragment, does what we want.

I remember that towncrier will create one list item per file

On my project, as a hack I have `482-1.feature.rst` as a way to add another item in the notes...but I think that we need better support here

I remember there were other disussions about this